src/submission/submission-entities.ts

```diff
@@ -297,16 +316,22 @@ export const BatchNameRegex: Record<ClinicalEntitySchemaNames, RegExp[]> = {
   [ClinicalEntitySchemaNames.REGISTRATION]: [/^sample_registration.*\.tsv$/],
   [ClinicalEntitySchemaNames.DONOR]: [/^donor.*\.tsv$/],
   [ClinicalEntitySchemaNames.SPECIMEN]: [/^specimen.*\.tsv$/],
-  [ClinicalEntitySchemaNames.PRIMARY_DIAGNOSIS]: [/^primary_diagnosis.*\.tsv/],
-  [ClinicalEntitySchemaNames.FOLLOW_UP]: [/^follow_up.*\.tsv/],
+  [ClinicalEntitySchemaNames.PRIMARY_DIAGNOSIS]: [/^primary_diagnosis.*\.tsv$/],
+  [ClinicalEntitySchemaNames.FOLLOW_UP]: [/^follow_up.*\.tsv$/],
+  [ClinicalEntitySchemaNames.TREATMENT]: [/^treatment.*\.tsv$/],
+  [ClinicalEntitySchemaNames.CHEMOTHERAPY]: [/^chemotherapy.*\.tsv$/],
 };
 
 // assumption: one field uniquely identifies a clinical type record in a batch of records
-export const ClinicalUniqueIndentifier = {
+export const ClinicalUniqueIndentifier: {
+  [clinicalType: string]: ClinicalFields;
+} = {
   [ClinicalEntitySchemaNames.DONOR]: DonorFieldsEnum.submitter_donor_id,
   [ClinicalEntitySchemaNames.SPECIMEN]: SpecimenFieldsEnum.submitter_specimen_id,
   [ClinicalEntitySchemaNames.PRIMARY_DIAGNOSIS]: DonorFieldsEnum.submitter_donor_id,
   [ClinicalEntitySchemaNames.FOLLOW_UP]: FollowupFieldsEnum.submitter_follow_up_id,
+  [ClinicalEntitySchemaNames.TREATMENT]: TreatmentFieldsEnum.submitter_treatment_id,
+  [ClinicalEntitySchemaNames.CHEMOTHERAPY]: TherapyFieldsEnum.chemotherapy_drug_name,
```

Comment:
I've just realised this won't really work with file submission because there can be multiple records using the same drug_name since it comes from a codelist.